src/main/java/com/java_essential/varycheva/essential/homework/hw8/task1/InputApp.java

```diff
@@ -0,0 +1,57 @@
+package com.java_essential.varycheva.essential.homework.hw8.task1;
+
+import java.util.Scanner;
+
+public class InputApp {
+
+    private final Scanner scanner = new Scanner(System.in);
```

Comment:
It's good idea, but try to avoid method shared resource declarations
```suggestion
    private final Scanner scanner = new Scanner(System.in);
```
```suggestion
   
```